solidity/contracts/utils/Ownable.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IOwnable} from '../../interfaces/utils/IOwnable.sol';
+
+/**
+ * @title  Ownable
+ * @notice This contract is used to manage ownership
+ */
+contract Ownable is IOwnable {
```

Comment:
how is this not OZ OwnableTwoStep? what is it better or changed?

Reply:
Just create this one because OZ uses context. I thought this was simpler and so the errors and events were under our project. I can use the other without any problem.